test/schema.test.ts

```diff
@@ -38,9 +38,10 @@ describe('Notices file is valid', () => {
       });
 
       test('v2 version ranges must be bounded at the bottom', () => {
+        const v2Range = '>=2.0.0-rc.1';
         for (const component of notice.components) {
           if (component.version === '1.*') { continue; } // Special range that we allow
-          if (semver.intersects(component.version, '2') && !semver.subset(component.version, '2')) {
```

Comment:
Using `2` as the range wouldn't allow pre releases, and we still have customers users those...

Reply:
```
> semver.satisfies('2.0.0-rc.2', '>=2.0.0-rc.1 <2.80.0')
true
> semver.satisfies('2.0.0-rc.2', '^2.0.0 <2.80.0')
false

```

Reply:
I think what we're trying to convey here is that:

* A range may not cross a major version boundary
* A range must be bounded at the top

Maybe it would be simpler to do the following:

* Do a linear scan of version ranges, test `x.0.0` and `x.10000.0` and make sure not more than one `x` is within the range
* Of that same `x`, test `x.10000.0` and test that it is not matched.

Reply:
Potentially, I honestly don't have the mental capacity right now to rework this...